### PR TITLE
Refresh token management on the identity-facade reference

### LINKED ISSUE #618: Refresh token management on the identity-facade reference
Refresh token management on the identity-facade reference

### PR DESCRIPTION
**Pull request by PratMoha**
#618

## Description
Added support for Refresh token management on the identity-facade reference flow. 

- Enhanced the conditions, and added policies for refresh token generation, set/extract token attributes, assign message request for refresh token generation.
- Added RFCCompliantRequestResponse to the token generation policy for compatibility with external OAuth plugins.

  - Error and non-error responses will include the HTTP Cache-Control response header field to comply with [RFC2616]
  - The expires_in property will be in alpha-numeric format. For consistency, the refresh_token_expires_in will also be alpha-numeric.
  - OAuthV2 responses for token generation, will include the RFC-compliant Bearer header field instead of BearerToken.
  - Error messages in response payloads will be of the format: {"error" : "xxx", "error_description" :"yyy"} for refresh token errors.

## Issues Fixed

- Fixes #618 

## Housekeeping
(please check all that apply [x], do not edit the text)
- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

## Full Repo Validation Required
(please check all that apply [x], do not edit the text)
- [ ] PR requires full pipeline run (Run for changes only by default).

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Review by JoelGauci — APPROVED** [review]

File: references/identity-facade/diagram/sequence-identity-facade-v1.png
Comment (PratMoha) [inline references/identity-facade/diagram/sequence-identity-facade-v1.png:1]: @JoelGauci @danistrebel Is there a way to generate the Sequence diagram in the previous format? The current file format doesn't have enough clarity.

Comment (danistrebel) [reply references/identity-facade/diagram/sequence-identity-facade-v1.png:1]: I think @JoelGauci used [PlantUML](https://plantuml.com/) according to [this script](https://github.com/apigee/devrel/blob/main/references/identity-facade/generate-docs.sh). Is that what you used too?

Comment (PratMoha) [reply references/identity-facade/diagram/sequence-identity-facade-v1.png:1]: I have used PlantUML as well.

Comment (JoelGauci) [reply references/identity-facade/diagram/sequence-identity-facade-v1.png:1]: I have not used the online image generator but downloaded a jar file (plantuml.1.2020.23.jar) to generate the image locally. This is the only difference I see and that might explain the difference of picture rendering

**Pull request merged**